docs/using-redis.md

```diff
@@ -0,0 +1,93 @@
+# Gluster Volume as persistent store for running Redis server
```

Comment:
New suggested title: "Using a Gluster volume for persistent application storage"